### PR TITLE
Add filtering on module to prevent warnings from other modules

### PR DESCRIPTION
**Pull request by oysteoh**

### CODE REVIEW

**Review by sondreso — APPROVED** [review]
LGTM! Tested locally :robot:

**Comment by sondreso** [comment]
Note: Related to https://github.com/equinor/libecl/issues/679

File: python/ert/__init__.py [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import warnings
 
-warnings.simplefilter('always', DeprecationWarning) # see #1437
```
Comment (oyvindeide) [inline python/ert/__init__.py:3]: Should probably do this globally then? There is another one in: libres/python/res/__init__.py

Comment (oyvindeide) [reply python/ert/__init__.py:3]: Never mind, didn't see that you'd already done that

**Pull request merged**